pkg/cli/version_test.go

```diff
@@ -0,0 +1,50 @@
+/*
+Copyright 2022 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package cli
+
+import (
+	"bytes"
+
+	. "github.com/onsi/ginkgo/v2"
+	. "github.com/onsi/gomega"
+	"github.com/spf13/cobra"
+)
+
+var _ = Describe("CLI", func() {
+	var cli CLI
+	var buf *bytes.Buffer
+
+	BeforeEach(func() {
+		buf = &bytes.Buffer{}
+		cli = CLI{commandName: "myapp", version: "1.0.0"}
+	})
+
+	Describe("newVersionCmd", func() {
+		var cmd *cobra.Command
+
+		BeforeEach(func() {
+			cmd = cli.newVersionCmd()
+			cmd.SetOutput(buf)
+		})
+
+		It("prints the version number", func() {
+			err := cmd.Execute()
+			Expect(err).ToNot(HaveOccurred())
+			Expect(buf.String()).To(Equal("1.0.0\n"))
```

Comment:
@khareyash05 it is failing 

<img width="772" alt="Screenshot 2023-04-22 at 19 41 27" src="https://user-images.githubusercontent.com/7708031/233801218-ca25fa07-2773-483a-80d5-90cf632067fa.png">

You can either run it locally to fix it out.